src/components/connectAdvanced.js

```diff
@@ -65,30 +43,33 @@ export default function connectAdvanced(
     // determines whether this HOC subscribes to store changes
     shouldHandleStateChanges = true,
 
-    // the key of props/context to get the store
+    // the key of props/context to get the store [**does nothing, use consumer**]
     storeKey = 'store',
 
     // if true, the wrapped element is exposed by this HOC via the getWrappedInstance() function.
     withRef = false,
 
+    // the context consumer to use
+    consumer = Consumer,
+
     // additional options are passed through to the selectorFactory
     ...connectOptions
   } = {}
 ) {
-  const subscriptionKey = storeKey + 'Subscription'
-  const version = hotReloadingVersion++
+  invariant(!withRef,
+    `withRef is removed. To access the wrapped instance, simply pass in ref`
+  )
 
-  const contextTypes = {
-    [storeKey]: storeShape,
-    [subscriptionKey]: subscriptionShape,
-  }
-  const childContextTypes = {
-    [subscriptionKey]: subscriptionShape,
-  }
+  invariant(storeKey === 'store',
+    'storeKey is deprecated and does not do anything. To use a custom redux store for a single component, ' +
```

Comment:
If it doesn't do anything, it's not deprecated :) I'd just say "The storeKey option has been removed."